tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Responses/GetOrderResponse.cs

```diff
@@ -43,6 +43,14 @@ public void AssertOrderItemCost(decimal orderItemCost)
             actual.Should().Be(orderItemCost);
         }
 
+        public void AssertOrderItemRecipientCost(string odsCode, decimal orderItemCost)
+        {
+            var item = ReadOrderItems(ContentAsJson).First(i => i.CatalogueItemName is not null);
+            ExpectedServiceRecipient serviceRecipient = item.ServiceRecipients.First(i => i.OdsCode == odsCode);
```

Comment:
If we make the change suggested above we could select the item based on the item ID rather than testing the name.

I'm not sure we gain anything by testing that the item name is not null – the `Name` column has a `NOT NULL` constraint so a catalogue item name should never be null.

Reply:
Scenario steps updated with CatalogueItemId.
If CatalogueItemName is NOT NULL then based on that I have also updated metho `AssertOrderItemCost` to get the actual total cost